docs/dev/docs.md

```diff
@@ -66,19 +49,41 @@ Use `mkdocs gh-deploy --help` for a full list of options available for the gh-de
  
 :fa-info-circle: _**Important:**_ Never manually edit files on the `gh-pages` branch because you will lose your work the next time you run the `gh-deploy` command.
 
+
 ## Contributing to Documentation
 
+:fa-info-circle: _**Important:**_ If you added a new markdown file to `docs/`, please make sure to include it in it relevant section in `mkdocs.yml` to make it accessible.
+
+
 We use github issues to keep track of hanging fruits to improve ShapeWorks documentation. For an internal list of todos, which will be turned to issues, visit [Documentation ToDo List](../todo.md).
```

Comment:
I've never heard "hanging fruits" used like this, just "low hanging fruits". I'd recommend changing the sentence to something like "We use GitHub to keep track of issues pertaining to ShapeWorks documentation."

Reply:
Done